EduPerfTests/Program.cs

```diff
@@ -179,17 +255,15 @@ static void DeterminePageLoadSites()
             }
 
             DeterminePageLoadSiteStart();
-            DeterminePageLoadIterations();
         }
-
+        
         private static void DeterminePageLoadSiteStart()
         {
-            int startingSite;
             Console.WriteLine("Specify the site number to start with. The first item is 1. (blank to run all sites):");
             var input = Console.ReadLine();
             if (string.IsNullOrWhiteSpace(input)) return;
 
-            startingSite = int.Parse(input);
+            int startingSite = int.Parse(input);
```

Comment:
If we're going with var in other places I'd argue we should go with var here as well. It's fairly obvious what int.Parse() will return to a user so calling it out doesn't do much.